Makefile

```diff
@@ -85,7 +85,10 @@ jetstream_config:
 	rm -Rf experimenter/experimenter/segments/metric-hub-main/.script/
 
 feature_manifests: build_dev
-	$(COMPOSE) run -e GITHUB_BEARER_TOKEN=$(GITHUB_BEARER_TOKEN) experimenter /experimenter/bin/manifest-tool.py fetch $(FETCH_ARGS)
+	$(COMPOSE) run \
+		--rm \
```

Comment:
If we're doing this pattern of

```
$(COMPOSE) run \
		--rm \
```

everywhere, could we just add it as another alias at the top of the file so we don't have to repeat it everywhere? 🤔